### DISCUSSION TITLE
dynamic_smagorinsky_coeffs

### DISCUSSION THREAD

**Discussion by bronise777** [AMReX-Combustion/PeleC#845] [General]
In the _**pc_dynamic_smagorinsky_coeffs**_ function, should _**M**_ here be the difference of the stress tensor instead of the product?
// "resolved turbulent stresses" and others
  for (int dir = 0; dir < AMREX_SPACEDIM * AMREX_SPACEDIM; dir++) {
    M[dir] = betaij[dir] * alphaij(i, j, k, dir);
  }

**Comment by marchdf** [comment]
Can you point to the line in a paper that is making you think this? You could be right.

**Comment by bronise777** [comment]
Formula 17 on page 365 in  Martín, M. Pino, U. Piomelli, and G. V. Candler. "Subgrid-Scale Models for Compressible Large-Eddy
Simulations." Theoretical and Computational Fluid Dynamics 13, no. 5 (2000): 361–76. https://link.springer.com/article/10.1007/PL00020896

**Reply by marchdf** [reply]
Yeah it definitely looks like a bug... I opened a PR to fix. Thank you!
